Logstash/logstash-input-azurewadtable/lib/logstash/inputs/azurewadtable.rb

```diff
@@ -52,14 +53,34 @@ def run(output_queue)
   def teardown
   end  
 
-  def process(output_queue)
-    @logger.debug(@last_timestamp)
+  def build_latent_query
```

Comment:
_partitionkey_from_datetime_ omits seconds from time. 

As a result, there still could be the case when data from the same minute will be lost:

1. _last_timestamp_ = 06:59, _data_latency_seconds_ = 1
2. VM-A wrote data at 07:00:00.000Z
3. logstash queried data at 07:00:14.000Z. _until_timestamp_ is 07:00:13.000Z. The query is: 06:59 < PartitionKey < 07:00. 
Got data from VM-A, moved _last_timestamp_ to 07:00 (considering there is no mode data and _continuation_token_ is null)
4. VM-B wrote data at 07:00:15.000Z
5. logstash queried data at 07:00:29.000Z. _until_timestamp_ is 07:00:28.000Z. The query is:  07:00 < PartitionKey < 07:00.

Data from VM-B is lost.

So:
1. _data_latency_seconds_ fraction should be minutes.
2. Is there a scenario when someone need more than 1 minute delay? Why would someone need to configure delay at all?